samples/drivers/spi_flash/boards/nrf52840_pca10056.conf

```diff
@@ -1,7 +1,12 @@
 #
-# Copyright (c) 2019 Peter Bigot Consulting, LLC
```

Comment:
rather add copyright, not replace

Reply:
This commit is actually from me, but authorship got lost during development, and I removed  this because in the churn around this I'd forgotten it was originally added on my own time not for Nordic.  But yes, it probably should have both, just for history.

I would rather that the commit was restored to the original authorship and sign-off, if that can be done.  `git commit --amend -C sha-of-original` should do it.